ugvc/utils/metrics_utils.py

```diff
@@ -163,10 +165,13 @@ def read_sorter_statistics_csv(sorter_stats_csv: str) -> pd.DataFrame:
         df_sorter_stats.loc["PCT_Failed_QC_reads"] = df_sorter_stats.loc["Failed_QC_reads"] / (
             df_sorter_stats.loc["Failed_QC_reads"] + df_sorter_stats.loc["PF_Barcode_reads"]
         )
-    # rename metrics to uniform convention
-    df_sorter_stats = df_sorter_stats.rename({c: c.replace("PCT_", "% ") for c in df_sorter_stats.index})
+
+    if edit_metric_names:
+        # rename metrics to uniform convention
+        df_sorter_stats = df_sorter_stats.rename({c: c.replace("PCT_", "% ") for c in df_sorter_stats.index})
     return df_sorter_stats
 
+
 def read_effective_coverage_from_sorter_json(
```

Comment:
Can you add type hints and docstring here please?

Reply:
That's Rinas